### ISSUE TITLE
180 degrees East / West bug

### ISSUE THREAD

**Issue by Snoopy316** [flybywiresim/a32nx#5592]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-08-07T04:01:01+00:00",
  "ref": "master",
  "sha": "10169b43abb8568044bab599783275b34f2bca70",
  "actor": "tracernz",
  "event_name": "manual"
}
```

### Describe the bug

When flying a pacific route which crosses the International date line (when 180 degrees east becomes 180 degrees west or vice versa) there is a display issue with the route where it will be drawn going off sideways (screenshot). In addition to this there is a big performance hit after crossing over, especially if altering the flight plan in any way like doing a direct or setting up a STAR. I suppose it's trying to compute a line all the way around the world or something similar. so the CPU completely chokes on it

This isn't specifically a FBW issue as I've had this in the 787 as well but if there's any way to fix that would be great :)

 
![PFD](https://user-images.githubusercontent.com/47849292/128638865-b1ed6806-fa9b-473a-a41f-7f0002a75b26.png)

### Expected behavior

Flight plan is drawn correctly as a continuous straight like when passing East - West or West - East at 180 degrees longitude

No performance / redraw issues after crossing the 180 degree point and or amending a flight plan. 

### Steps to reproduce

pick any route which crosses 180 east / west (pacific island hopping like NFFN / NSTU is probably quickest way to get to it)

you should see the display issue. And then amending the flight plan in flight (doing directs, entering STARS etc.. anything that prompts a redraw of the flight plan) should trigger the performance issues.

### Discord Username (optional)

TheOnlyMilhouse

**Comment by Snoopy316** [comment]
Forgot to add - the route being flown in the screenshot is this one:

NZAA/23L MEMO1P SELKA B575 TUT DCT NSTU/05

**Comment by tracernz** [comment]
It would be interesting to check this on the new ND in the custom fpm branch…

**Comment by hiroooo1997** [comment]
It's a [msfs bug](https://forums.flightsimulator.com/t/big-and-old-problem-with-the-flight-going-far-east-to-far-west-international-date-line/346829) since day 1.

**Comment by Snoopy316** [comment]
I see. It doesn't look like there are many votes on there for getting it fixed which is a shame. I suppose not many routes are affected by it.

I've noticed some other things as well, like the aircraft that was ahead of me in that screenshot disappeared when it crossed the line and it didn't reappear until I crossed over as well. Same with displaying airports and waypoints. It's like nothing in the two East West hemispheres can co-exist

It would be cool if there was a fix for the FBW but yea, looks like Asobo have done something profound with how the world is stitched together

**Comment by davidwalschots** [comment]
It might be because the world is flat and when you cross there you are in fact teleported to the other side!

**Comment by hiroooo1997** [comment]
"autopilot-custom-fpm" branch doesn't have this problem and draws collect line over 
180 degrees East / West in ND, but VFR map line seems unavailable for now,

**Comment by 2hwk** [comment]
Fixed by #4953 closing ☑️
